### DISCUSSION TITLE
[New Feature] Allow player to see friendly cloaked objects in single-player

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1481] [Feature Requests]
### Description:
In single-player missions, any cloaked objects not belonging to the player's House (`[Basic]`->`Player=`) will be invisible and unselectable to the player, even if those objects belong to a House that's allied to the player and/or has `PlayerControl=` set. This behavior has its pros and cons for mission design. It would be best if this could be toggled.

### Possible Implementation:
New General/AudioVisual key, e.g. `CloakVision=`, which, if set, shows friendly cloaked objects as semi-transparent to the player. Or maybe have this as a House key.

**Comment by JoyfulShush** [comment]
In addition to this, players cannot select allied units that are cloaked even though they can see them until they uncloak; might be good to also allow this as well.
